### DISCUSSION TITLE
Keybind go to specific tab

### DISCUSSION THREAD

**Discussion by gabrielgrs** [zed-industries/zed#21483] [Feature Requests]
I am trying to leave vscode and start using Zed, but I am feeling bad trying to configure this because I am taking a lot of time to configure simple things.

First question, is there any tutorial to help me with zed basics?

Second, how to configure keybind to work on with cmd and number to go to specific Tab, for example, cmd+1 to go to the first tab, cmd+2 to go to the second tab, etc

Third, how to disable some keybind? 

Thanks!

**Comment by volkan0x** [comment]
`cmd-k` + `cmd-s` to open the keymap.json file and then delete disable the whatever keybind you want. I'd recommend go to the "context": "pane" section in the keymap.json file and use the tab switching like this. This is way more productive, imo.:


"context": "Pane",
    "bindings": {
      "alt-1": "pane::ActivatePrevItem",
      "alt-2": "pane::ActivateNextItem",

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
